src/appsettings/settings.py

```diff
@@ -837,6 +893,21 @@ def __init__(
         if max_length is not None:
             self.validators.append(MaxLengthValidator(max_length))
 
+    def decode_environ(self, value):
+        """
+        Split value by colons to a list.
+
+        Args:
+            value (str):
+
+        Returns:
+            list:
+        """
+        split_value = value.split(":")
```

Comment:
`:` is not the best separator if you are dealing with urls, for example. I'd suggest different separator with a possibility of escaping.

Reply:
I have implemented @ziima 's idea:

> `ListSetting` could use a `:` as a separator, similarily to `PYTHONPATH`.

But I agree that `:` is quite common literal. Which separator would you recommend then? Or should I try to implement an escape posibility for `:`?

Reply:
For URIs, the best separator would be a newline I guess. So maybe you could first try to split on `"\n"`, and if there's only one item in the result, fallback to split on `":"`, which is the most common and expected separator.

But then a valid list of one URI wouldn't be handled correctly...

Reply:
I think that `:` is ok as long as you can escape it.

Reply:
All right, I made it that way. After splitting the value i replaced the escaped colons. Hope it works.